test/e2e/serial/tests/configuration.go

```diff
@@ -1018,7 +1026,5 @@ func waitForMcpsCondition(cli client.Client, ctx context.Context, mcps []*machin
 			return err
 		})
 	}
-	if err := eg.Wait(); err != nil {
-		fmt.Printf("An error occurred: %v\n", err)
-	}
```

Comment:
yet another reminder that swallowing errors is almost always a bad idea, and that some italian reviewer should be stricter about it

Reply:
please allow me to take responsibility for that, the goal for moving from wait group to errorgroup was mainly for handling the error properly, it really was a bad implementation for the intended new alternative just reporting the error